pages/awards.js

```diff
@@ -1,55 +1,30 @@
+import AwardsJSON from '../assets/data/awards.json';
+import { Container, Row, Col } from 'react-bootstrap';
+
+function Section(props) {
+  const {title, provider} = props;
+  return (
+    <Col md={6}>
+      <h5>{props.title}</h5>
+      <p>{props.provider}</p>
+    </Col>
+  )
+}
+
 export default function Awards() {
   return (
     <>
-      <h1 class="display-sm-4 display-lg-3">Awards & Recognitions</h1>
-
-      <main role="main">
-        <section class="text-center bg-light u-content-space">
-          <div class="container">
-            <header class="text-center w-md-50 mx-auto mb-8">
-              <h2 class="h1">Awards</h2>
-            </header>
-
-            <div class="row">
-              <div class="col-lg-6 mb-5 mb-lg-0">
-                <h4 class="h5">Most Outstanding Social/Support Organization</h4>
-                <p class="mb-0">Anteater Awards 2011</p>
-              </div>
-
-              <div class="col-lg-6 mb-5 mb-lg-0">
-                <h4 class="h5">Student Alumni Engagement Sponsorship</h4>
-                <p class="mb-0">Student Alumni Association</p>
-              </div>
-            </div>
-          </div>
-        </section>
-
-        <section class="u-content-space">
-          <div class="container">
-            <header class="text-center w-md-50 mx-auto mb-8">
-              <h2 class="h1">Recognitions</h2>
-            </header>
-
-            <div class="row">
-              <div class="col-lg-6 text-center pl-lg-5">
-                <h2>2020</h2>
-              </div>
-
-              <div class="col-lg-6 text-center text-lg-left pl-lg-5">
-                <ul>
-                  <li>
-                    <p>3rd Place in Winter UCI Alumni Association Club Affiliates Challenge</p>
-                  </li>
-                  <li>
-                    <p>Second place in UCI Campus Organizations & Volunteer Programs I Love My Org contest</p>
-                  </li>
-                </ul>
-              </div>
-            </div>
-          </div>
-        </section>
-
-      </main>
+      <h1>Awards & Recognitions</h1>
+      <Container className="text-center" role="main">
+        <h2>Awards</h2>
+        <Row>
+          {AwardsJSON.map(awardObj => awardObj["awards"].map(award => <Section {...award}/>))}
```

Comment:
Now that awards is just a key on the json, we can simplify this
```suggestion
          {AwardsJSON["awards"].map(award => <Section {...award}/>)}
```

Reply:
Committed the suggestions! How is it now?